src/apis/TurboModule.res

```diff
@@ -0,0 +1,9 @@
+type turboModule = {@optional getConstants: option<unit => Js.Dict.t<Js.Json.t>>}
```

Comment:
I don't think we need this type.

If the implementer needs to get constants from his module, it will be easier if `getConstants` isn't an option, and if it's a record instead of a dict.